### PR TITLE
Fixes from code review

### PR DESCRIPTION
**Pull request by collinsugwu**
what this PR does
1 use html semantics
2 add search bar to input
3 adjust the nav bar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -23,69 +25,73 @@
...
-                            <li><a href="#"><img src="resources/img/usa.png" alt="usa"></a></li>
-                        </ul>
-                    </div>
+            </div>
+        </section>
+        <section class="product-section">
+            <ul>
+                <li>
+                    <figure class="product-photo">
+                        <img src="resources/img/iphone5s.jpg" alt="iPhone5s">
```
Comment (Ispirett) [inline index.html]: These images are actually links so they should be wrapped in `<a>` tags

**Review by Ispirett — CHANGES_REQUESTED** [review]
Great Job @collinsugwu  you did excellent work with this project and changes. :+1: :100: 

**Some changes to make are below**

Please add the the the`<meta charset="UTF-8">

Submit for rereview when finished

File: resources/css/main.css [hunk 2]
```diff
@@ -39,6 +39,16 @@ ul li {
...
+  position: relative;
+}
+
+.search {
+  position: absolute;
+  top: 20%;
+  left: 2%;
+}
+
 .header {
```
Comment (whiz25) [inline resources/css/main.css:52]: Your ```nav``` doesn't quite have the ```gradient``` styling which is the main objective of this project. Try adding ```background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, .2)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .45))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, .2)), color-stop(0.002, rgba(0, 0, 0, 0)), color-stop(0.998, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, .2)))``` styling to the background of your ```nav```.

**Review by whiz25 — CHANGES_REQUESTED** [review]
## Good job :+1: @collinsugwu 
This project is almost complete. But it requires some changes to be made.
- [ ] Style the ```navbar``` background using ```gradient``` just like the original site. Please see my comment.

## [NEXT PROJECT]
- [ ] Please follow the [github flow](https://guides.github.com/introduction/flow/) in your next projects.

Make those changes and send for another review.

**Review by wathigo — APPROVED** [review]
### Great work Guys :clap: 
**You may move on :)**
_Good luck in your future projects_

**Pull request merged**